docs/docs/troubleshooting.md

```diff
@@ -40,6 +40,10 @@ West build errors usually indicate syntax problems in the `<keyboard>.keymap` fi
 If you are reviewing these errors in the GitHub Actions tab, they can be found in the `West Build` step of the build process.
 :::
 
+#### Keymap error
+
+If you get an error stating `Keymap node not found, check a keymap is available and is has compatible = "zmk,keymap" set` this is an indication that the build process cannot find the keymap. Double check that the `<keyboard>.keymap` file is present and has been discovered by the build process. This can be checked by looking for a line in the build log stating `-- Using keymap file: /path/to/keymap/file/<keyboard>.keymap`. Inside the keymap file ensure the keymap node has `compatible = zmk,keymap` and it's not misspelled. For more information see the [Keymap](features/keymaps.mdx) documentation.
```

Comment:
If the main reason we think users get this is that the keymap file isn't found, we can also link to https://zmk.dev/docs/config in addition to the Keymap page.

Reply:
Added a link to the config documentation in addition to the keymap link that's currently there